optaplanner-docs/src/main/docbook/en-US/Chapter-Planner_configuration/Chapter-Planner_configuration.xml

```diff
@@ -253,8 +253,8 @@
       </note>
 
       <para><emphasis role="bold">In Planner all problems facts and planning entities are plain old JavaBeans
-      (POJO's).</emphasis> You can load them from a database, an XML file, a data repository, a noSQL cloud, ... (see
-      <link linkend="integration">Integration</link>): Planner doesn't care.</para>
+      (POJOs).</emphasis> You can load them from a database, an XML file, a data repository, a noSQL cloud, ... (see
+      <link linkend="integration">Integration</link>).</para>
```

Comment:
About "Planner doesn't care": The way it's stated is probably too negatively/unproffesionally, but it is an important checkbox in people's head: the fact that optaplanner doesn't care if those POJO's come from XML or from a relational database or whatsoever. It means it doesn't matter which what it is combined, there are no pitfalls when integrating with any technology as long as that technology is able to deliver POJO's.

I'll merge this PR (so delete the original statement), but if you see anyway to describe the above sentiment in a good way, it's welcome :)